config/gregtech/worldgen/vein/nether/sulfur_deposit_vein.json

```diff
@@ -0,0 +1,45 @@
+{
+  "weight": 120,
+  "density": 1,
+  "dimension_filter":[
+    "dimension_id:-1"
+  ],
+  "max_height":120,
+  "min_height":0,
+  "generator": {
+    "type": "layered",
+    "radius": [
```

Comment:
this may be a little large, remember this would generate a 40 block radius area of pure sulfur